DeepMRSeg/deepmrseg_test.py

```diff
@@ -432,140 +461,387 @@ def _main():
 		nJobs = _os.cpu_count()
 	#ENDIF
 
-	# if otherMods defined
+	# Create temp dir
 	#IF
-	if FLAGS.otherMods:
-		otherMods = list(map( str, FLAGS.otherMods.split(',') ))
-		num_modalities = len(otherMods) + 1
+	if FLAGS.tmpDir:
+		user_defined_tmpdir = True
+		tmpDir = FLAGS.tmpDir
+		if not _os.path.isdir(FLAGS.tmpDir):
+			_os.makedirs( FLAGS.tmpDir )
 	else:
-		otherMods = None
-		num_modalities = 1
+		user_defined_tmpdir = False
+		tmpDir = _tempfile.mkdtemp( prefix='deepmrseg_test_' )
 	#ENDIF
 
+	print("Temp folder set to : " + tmpDir)
+	_sys.stdout.flush()
+	
+
 	### Print parsed args
-	print( "\nFile List \t: %s" % (FLAGS.sList) )
-	print( "ID Column \t: %s" % (FLAGS.idCol) )
-	print( "Output Column \t: %s" % (FLAGS.outCol) )
-	print( "Ref Modality \t: %s" % (FLAGS.refMod) )
-	print( "Other Mods \t: %s" % (otherMods) )
-	print( "Num of Mods \t: %d" % (num_modalities) )
+	if IOType == 1:
+		print( "\nInput image(s) \t: %s" % (' , '.join(FLAGS.inImg)) )
```

Comment:
Lets print all paths as abspath. This needs changing for some of the other args as well.

Reply:
Here I type whatever input is entered by the user. I convert them to abs path at a later time. We can print after.